bindings/node/test/clientEncryption.test.js

```diff
@@ -815,4 +828,220 @@ describe('ClientEncryption', function () {
   it('should provide the libmongocrypt version', function () {
     expect(ClientEncryption.libmongocryptVersion).to.be.a('string');
   });
+
+  describe('createEncryptedCollection()', () => {
+    /** @type {InstanceType<ClientEncryption>} */
+    let clientEncryption;
+    const client = new MockClient();
+    let db;
+    const collectionName = 'secure';
+
+    beforeEach(async function () {
+      clientEncryption = new ClientEncryption(client, {
+        keyVaultNamespace: 'client.encryption',
+        kmsProviders: { local: { key: Buffer.alloc(96, 0) } }
+      });
+
+      db = client.db('createEncryptedCollectionDb');
+    });
+
+    afterEach(async () => {
+      sinon.restore();
+    });
+
+    context('validates input', () => {
+      it('throws if options are omitted', async () => {
```

Comment:
Let's be specific about what errors these throw.  For example, `throws TypeError if options are omitted` (although I suggest above that we throw a `MongoCryptInvalidArgumentError`).

Reply:
added TypeError